usr/bin/dhcpd-update

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/python
```

Comment:
Why making this file compatible python2-3?
You could juste convert it to python3